packages/telestion-client-common/src/components/header/dashboard-picker/use-dashboard-state.ts

```diff
@@ -7,30 +7,120 @@ import { AuthState, useAuth } from '@wuespace/telestion-client-core';
 // auth selector
 const selector: StateSelector<AuthState, boolean> = ({ auth }) => !auth;
 
+/**
+ * The key of a picker element which represents a dashboard in the picker.
+ */
 export type DashboardPickerKey = string;
 
+/**
+ * The item of a dashboard picker.
+ * It has a title that is shown on selection
+ * and a key of the dashboard to show
+ * when the user presses the picker element.
+ */
 export interface Item {
+	/**
+	 * The dashboard key the picker item refer to.
+	 */
 	key: DashboardPickerKey;
+
+	/**
+	 * The title to show if the dashboard is active.
+	 * It is usually the title of the dashboard.
+	 */
 	title: string;
 }
 
-export interface SelectState {
+/**
+ * The current state of the dashboard picker.
+ * It is a return type of the {@link useDashboardState} hook
+ * which controls the behaviour of the dashboard picker.
+ */
+export interface PickerState {
+	/**
+	 * The picker elements the dashboard picker should render.
+	 *
+	 * @see {@link Item}
+	 */
 	items: Array<Item>;
 
+	/**
+	 * The picker element where the {@link Item.key}
+	 * matches the currently active dashboard.
+	 */
 	selected: DashboardPickerKey;
 
+	/**
+	 * A handler that changes the currently active picker element
+	 * and the application path to the new dashboard.
+	 * The application typically will then render the dashboard
+	 * with the specified {@link Item.key}.
+	 *
+	 * @param key - the key of the pressed item
+	 */
 	onSelectionChange: (key: ReactText) => void;
 
+	/**
+	 * If `true` the dashboard picker should be disabled
+	 * and show the placeholder text
+	 * because there are no registered dashboards.
```

Comment:
```suggestion
	 * If `true`, the dashboard picker should be disabled
	 * and show the placeholder text
	 * because there are no registered dashboards.
```